### DISCUSSION TITLE
Does DB Push on Prisma replace or precede Migrate on PlanetScale?

### DISCUSSION THREAD

**Discussion by dakdevs** [planetscale/discussion#360] [Q&A] [answered]
I saw that there is a feature in PlanetScale to mark the migrations table:
![image](https://user-images.githubusercontent.com/12757641/209888491-7102050b-3ea6-41a1-85c0-c9cb5bec4e26.png)

Does this mean that we still run our `prisma migrate deploy` command, just after the `prisma db push` command?

If not then what is this feature for if we are not supposed to be doing migrations.

**Comment by jgreet** [comment, marked as answer]
`prisma db push` should be used instead of `prisma migrate` at this point. When using `db push` there's need to enable the copying of migration data.

For more see https://planetscale.com/docs/tutorials/automatic-prisma-migrations, in particular [Quick introduction to Prisma's db push](https://planetscale.com/docs/tutorials/automatic-prisma-migrations#quick-introduction-to-prismas-db-push) and the note at the top.
